DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1653,6 +1653,7 @@ private void MapSharer_MapDownloadComplete(object sender, SHA1EventArgs e) =>
         private void MapSharer_HandleMapDownloadComplete(SHA1EventArgs e)
         {
             string mapFileName = MapSharer.GetMapFileName(e.SHA1, e.MapName);
```

Comment:
Are you sure you shouldn't modify `GetMapFileName` instead?

Reply:
Thank you for advice. Check 01975a6ed9c5310a8b2cfd9302267da2405c5c4d.